### DISCUSSION TITLE
取代鋰電池，有毒外來種植物成鈉離子電池陽極

### DISCUSSION THREAD

**Discussion by Zoe06060606** [1101-GS4719-C/Python-Community-GS4719-C#797] [General]
鋰電池需求增高，但原物料主要來自幾個國家，同時價格也越來越高。
   現今科學家不斷研究鈉離子電池的可能性，原因是取代鋰離子電池以及鈉在地殼含量較多。
   大豕草是俄羅斯之入侵種，其汁液有毒，人類若沾到此汁液又曬到陽光會起泡，可能會灼傷或失明。俄羅斯國立大學和斯科爾科沃科學技術學院團隊透過大豕草而製成的硬碳陽極，電池庫倫效率達到87%，儲電容量雖然略低。大豕草會在夏天盛開，同時，夏天表現比冬天還要好。
   鈉離子電池應用未來可能會更廣泛，電池也會較便宜，雖然重量較重，但在工業應用或是補助再生能源不會有什麼大問題。

之前就有看到過鋰金屬因為需求太多可能會在近幾年就不夠用的資訊，現在越來越多鈉離子電池的替代方案，希望能越來越普及，技術越來越成熟，早日可以應用到日常生活。
[鈉離子電池新解法](https://tw.news.yahoo.com/鈉離子電池新解方-有毒外來種植物成電池陽極-020119459.html)

**Comment by WEI-CHEN-CHUN** [comment]
所以鈉離子電池和大豕草兩者之間是什麼關係?

**Reply by Zoe06060606** [reply]
用大豕草製成的硬碳陽極用於鈉離子電池的陽極

**Comment by moonnight3503** [comment]
把有毒的外來物種轉換成電池的內容物，不但可以減少被傷害的機率，也可對自然原生態做保護，我覺得這個想法很棒!

**Comment by Pietra1226** [comment]
科技真的太厲害了，把對生態有害的外來種轉換成可用能源，希望不要再對本身的生態造成破壞，像這種方式就很好。

**Comment by evanchou1204** [comment]
普及到家庭的部分可能真的要技術更進步才有辦法，但我覺得目前能用在工業或輔助已經很棒了！把有害的外來種做成有用的電池真的很讚！

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
這想法真的很有趣 對於將來也會有很大的幫助 運用有毒的物質創造價值 真是不錯的概念
